client/src/app/__tests__/TabsProviderSpec.js

```diff
@@ -424,17 +424,69 @@ describe('TabsProvider', function() {
     expect(providers['empty']).to.exist;
   });
 
+  describe('#getProviderNames', function() {
 
-  it('#getProviderNames', function() {
+    it('should return default provider names', function() {
 
-    // given
-    const tabsProvider = new TabsProvider();
+      // given
+      Flags.init({});
+      const tabsProvider = new TabsProvider();
 
-    // when
-    const providerNames = tabsProvider.getProviderNames();
+      // when
+      const providerNames = tabsProvider.getProviderNames();
 
-    // then
-    expect(providerNames).to.eql([ 'BPMN', 'CMMN', 'DMN', 'FORM' ]);
+      // then
+      expect(providerNames).to.eql(['BPMN', 'DMN', 'FORM']);
+
+    });
+
+    it('should return all provider names', function() {
```

Comment:
Minor thing: We (try) to structure our tests in a way to keep _two_ newlines between each `it` and `describe` block. So

```js
it('getProviders');


describe('#getProviderNames', function()) {
  
  it('should return default provider names');
  
  
  it('should return all provider names');
  
  
  it('should return cloud provider names only');
  
  
  it('should return platform provider names only');
  
});
```

Seems like the newline was even missed in the previous state, so let's get it clean here 🙂